docs/UserGuide.md

```diff
@@ -153,6 +171,17 @@ After running an opencom command e.g `opencom 7`, you should be switched to the
 
 ![opencom](images/opencom.png)
 
+
+### Viewing all commissions: `allcom`
+
+Displays all commissions across all customers in ArtBuddy.
+
+Format: `allcom`
+
+After running `allcom`, you should be switched to the commissions tab to view all commissions.
+To view the commissions for a specific customer, return to the customer list ([`opencom`](#opencom)) and select the customer from the list ([`opencus INDEX`](#opencus)).
```

Comment:
```suggestion
To view the commissions for a specific customer, return to the customer list ([`opencus`](#opencus)) and select the customer from the list ([`opencus INDEX`](#opencus)).
```

Reply:
fixed in d1cd747de8652954725f8a1173f1d9a330593240